vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.cpp

```diff
@@ -199,6 +200,17 @@ void ReactInstanceWin::Initialize() noexcept {
               std::move(m_appTheme),
               m_legacyReactInstance);
 
+          cxxModules.emplace_back(
+              Microsoft::ReactNative::DevSettingsModule::name,
+              [weakReactHost = strongThis->m_weakReactHost]() {
+                return std::make_unique<Microsoft::ReactNative::DevSettingsModule>([weakReactHost]() {
```

Comment:
>) [](start = 99, length = 1)

It must be 'noexcept' for VoidFunctor.

Reply:
How come it didn't yell at me?

Reply:
I would expect it to give a deprecated warning. Though, I may be using a wrong one. I should replace it in this version of Mso with static_assert.